plugins-infra.ci.jenkins.io.txt

```diff
@@ -136,19 +136,19 @@ ssh-agent:346.vda_a_c4f2c8e50
 ssh-credentials:308.ve4497b_ccd8f4
 sshd:3.322.v159e91f6a_550
 structs:337.v1b_04ea_4df7c8
-support-core:1375.va_256158e8881
+support-core:1386.vc7f0a_46f9e6c
 theme-manager:215.vc1ff18d67920
 timestamper:1.26
 token-macro:400.v35420b_922dcb_
 toolenv:1.2
 trilead-api:2.133.vfb_8a_7b_9c5dd1
 variant:60.v7290fc0eb_b_cd
-warnings-ng:10.7.0
+warnings-ng:11.1.0
```

Comment:
```suggestion
warnings-ng:10.7.0
```
Keeping current one as there are breaking changes since https://github.com/jenkinsci/warnings-ng-plugin/releases/tag/v11.0.0 to be evaluated.